### PR TITLE
Fix to convert ServerName to lowercase in SqlAzureDacPacDeploymentV1

### PR DESCRIPTION
**Pull request by EshitaD**
**Task name**: SqlAzureDacPacDeploymentV1

**Description**: This code change converts server names from uppercase to lowercase.

**Documentation changes required:** (Y/N) <Please mark if documentation changes are required>

**Added unit tests:** (Y/N) <Please mark if unit tests were added or updated according changes>

**Attached related issue:** [DTS TASK 2186002](https://dev.azure.com/mseng/AzureDevOps/_workitems/edit/2186002)

**Checklist**:
- [ ] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [ ] Checked that applied changes work as expected

### CODE REVIEW

File: Tasks/Common/VstsAzureRestHelpers_/VstsAzureRestHelpers_.psm1 [hunk 1]
```diff
@@ -943,6 +939,8 @@ function Get-AzureSqlDatabaseServerResourceId {
         $ResourceDetails = Invoke-RestMethod -Uri $uri -Method $method -Headers $headers -ContentType $script:jsonContentType
         foreach ($resourceDetail in $ResourceDetails.Value) {
             if ($resourceDetail.name -eq $serverName -and $resourceDetail.type -eq $serverType) {
+                $serverNameLower = $serverName.ToLower() 
+                $resourcedetail.id = $resourcedetail.id.Replace($serverName, $serverNameLower)
                 return $resourceDetail.id
```
Comment (z00sts) [inline Tasks/Common/VstsAzureRestHelpers_/VstsAzureRestHelpers_.psm1:944]: Can you explain the fix in details? Why this change fixes the problem

Comment (z00sts) [reply Tasks/Common/VstsAzureRestHelpers_/VstsAzureRestHelpers_.psm1:944]: I get the general idea, I'm talking about previous implementation: what was wrong with it?

Comment (EshitaD) [reply Tasks/Common/VstsAzureRestHelpers_/VstsAzureRestHelpers_.psm1:944]: ResourceDetail.Id consists of the servername which should be converted to lowercase, the same we implemented here.
But in the previous change we converted the servername, which wont get reflected to the servername in resourcedetail.id.

Comment (z00sts) [reply Tasks/Common/VstsAzureRestHelpers_/VstsAzureRestHelpers_.psm1:944]: LGTM
